package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "reims-js-2103-project3-personality-back",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "start": "nodemon server.js"
```

Comment:
keep start script with node command: once deployed, you won't run nodemon. nodemon should be used in a dev script